cylc/flow/network/resolvers.py

```diff
@@ -59,6 +60,14 @@
     from cylc.flow.workflow_status import StopMode
 
 
+class TaskMsg(NamedTuple):
+    """Tuple for Scheduler.message_queue"""
+    task_job: Union[Tokens, str]
```

Comment:
Is this "task or job" or "job [associated with task]"?

FYI I'd like to retire the term "task-job" in favour of either "task" or "job".

https://github.com/cylc/cylc-doc/issues/352

Reply:
Inspected the variable in the debugger and it is just the job ID e.g. `1/foo/01`. Will push a change